dev/manual_tests/card_collection.dart

```diff
@@ -324,7 +324,7 @@ class CardCollectionState extends State<CardCollection> {
         child: new SingleChildScrollView(
           child: new Container(
             height: cardModel.height,
-            decoration: new BoxDecoration(backgroundColor: theme.primaryColor),
+            color: theme.primaryColor,
```

Comment:
That might be clearer.  I sort of regret using the `backgroundColor` name in `BoxDecoration`.  We did that to match CSS `background-color`, but it's confusing to have a `backgroundColor` for your `foregroundDecoration`...